pkg/queue/concurrency_state.go

```diff
@@ -22,20 +22,31 @@ import (
 	"net/http"
 	"os"
 	"sync"
+	"time"
 
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
 )
 
 //nolint:gosec // Filepath, not hardcoded credentials
 const ConcurrencyStateTokenVolumeMountPath = "/var/run/secrets/tokens"
+const FreezeMaxRetryTimes = 3 // If pause/resume failed 3 times, it should kill qp and delete user-container force
+
+// error code for pause/resume operation
+const (
+	noError                        = 0 // everything works
```

Comment:
let's use proper go `errors` instead of exit codes

Reply:
Yeah, it's better idea.